ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -848,7 +848,7 @@ tests:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-virtualmedia
 - as: e2e-openstack-ovn
-  run_if_changed: openstack|(^data/.*/rhcos\.json)
+  run_if_changed: (^(pkg|data|images|upi|cluster-api|vendor)/.*openstack)|(^data/.*/rhcos\.json)
```

Comment:
I understand the intent, but aren't we at risk of NOT running the CI on important changes if the structure of the installer changes?

Reply:
I'm leaning towards rejecting the change because the gain is minimal, for the rare changes to openstack docs, and introduces a risk of missing CI feedback on important changes. I'll let @stephenfin weight in.